app/src/main/java/org/kinecosystem/kinit/view/MainActivity.java

```diff
@@ -111,5 +156,18 @@ private void showPhoneVerifyPopup() {
         analytics.logEvent(new Events.Analytics.ViewPhoneAuthPopup());
     }
 
+    @Override
+    public void onAnimationStart(Animation animation) {
```

Comment:
it's hard to understand for what animation this listener is related, maybe you should create an anonymous listener on creation time. Or keep an instance for this listener instead of having the MainActivity implement it.